pkg/query-service/model/v3/v3.go

```diff
@@ -500,6 +509,7 @@ type BuilderQuery struct {
 	SelectColumns      []AttributeKey    `json:"selectColumns,omitempty"`
 	TimeAggregation    TimeAggregation   `json:"timeAggregation,omitempty"`
 	SpaceAggregation   SpaceAggregation  `json:"spaceAggregation,omitempty"`
+	Quantile           float64           `json:"quantile,omitempty"`
```

Comment:
I am not sure if it's wise to keep this field at toplevel. It seems like it should belong to AggreagtorOperator but I understand that would mean changing `AggreagtorOperator` to struct. I see you planning to use this field as a special case in your calculations too. Is that right?

Reply:
>How would you model this differently?

```type AggreagtorOperator  struct {
 type: string, // sum, mix, max, percentile
 value: null // expect for percentile
}
```
The advantage is scoping percentile to AggregateOperator. Any validation, restriction, etc has to be done inside AggregateOperator for percentiles too

Reply:
Using separate `TimeAggregation` and `SpaceAggregation` for metrics